src/tests.rs

```diff
@@ -1,449 +1,424 @@
-#[cfg(test)]
-mod tests {
-	use crate::polkadot_relay::runtime_types::frame_system::pallet::Call as PolkadotRelaySystemCall;
-	use crate::{
-		submit_referendum::{generate_calls, get_proposal_bytes},
-		CallInfo, CallOrHash, KusamaOpenGovOrigin, Network, NetworkRuntimeCall,
-		PolkadotOpenGovOrigin, PolkadotRuntimeCall, ProposalDetails, Weight,
-	};
-
-	fn polkadot_whitelist_remark_user_input() -> ProposalDetails {
-		use crate::DispatchTimeWrapper::*;
-		use crate::NetworkTrack::*;
-		use crate::Output::*;
-		return ProposalDetails {
-			// `system.remark("opengov-submit test")`
-			proposal: String::from("0x00004c6f70656e676f762d7375626d69742074657374"),
-			track: Polkadot(PolkadotOpenGovOrigin::WhitelistedCaller),
-			dispatch: After(10),
-			output: AppsUiLink,
-			output_len_limit: 1_000,
-			print_batch: true,
-			transact_weight_override: Some(Weight {
-				ref_time: 1_000_000_000,
-				proof_size: 1_000_000,
-			}),
-		}
+use crate::polkadot_relay::runtime_types::frame_system::pallet::Call as PolkadotRelaySystemCall;
```

Comment:
bad diff. just removes `mod tests`, because the file already creates a module called `tests` for this code.